code/modules/power/power_exporter.dm

```diff
@@ -0,0 +1,160 @@
+	icon_state = "yellow"
+	item_state = "yellow"
+	item_color="yellow"
+	permeability_coefficient = 0.05
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = HANDS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	resistance_flags = 0
+
+/obj/machinery/power/exporter/attackby(obj/item/O, mob/user, params)
+	if(!active)
+		if(istype(O, /obj/item/weapon/wrench))
+			if(!anchored && !isinspace())
+				connect_to_network()
+				user << "<span class='notice'>You secure the [src] to the floor.</span>"
+				anchored = TRUE
+			else if(anchored)
+				disconnect_from_network()
+				user << "<span class='notice'>You unsecure and disconnect the [src].</span>"
+				anchored = FALSE
+			playsound(src.loc, 'sound/items/Deconstruct.ogg', 50, 1)
+			return
+	return ..()
+
+/obj/machinery/power/exporter/attack_hand(mob/user)
+	..()
+	if (!anchored)
+		user << "<span class='warning'>This device must be anchored by a wrench!</span>"
+		return
+	if (power_drained >= 4000000 && !rewarded) // 4 billion watts
+		rewarded = TRUE
+		visible_message("<span class='danger'>You have been blessed by the gods of engineering for your massive power exports!</span>")
+		new /obj/effect/overlay/temp/explosion/fast(get_turf(src))
+		playsound(src, 'sound/effects/pray_chaplain.ogg', 100, 1)
+		for(var/i in 1 to 3)
+			new /obj/item/clothing/head/orange_beret(get_turf(src))
+			new /obj/item/clothing/suit/poncho/engiponcho(get_turf(src))
+			new /obj/item/clothing/gloves/krav_maga/engi(get_turf(src))
+	interact(user)
+
+/obj/machinery/power/exporter/attack_ai(mob/user)
+	interact(user)
+
+/obj/machinery/power/exporter/attack_paw(mob/user)
+	interact(user)
+
+/obj/machinery/power/exporter/interact(mob/user)
+	if (!anchored)
+		user << "<span class='warning'>This device must be anchored by a wrench!</span>"
+		user.unset_machine()
+		user << browse(null, "window=port_gen")
+		return
+	if(!Adjacent(user))
+		if(!isAI(user))
+			user.unset_machine()
+			user << browse(null, "window=port_gen")
+			return
+	user.set_machine(src)
+	var/dat = text("<b>[name]</b><br>")
+	if (active)
+		dat += text("Exporter: <A href='?src=\ref[src];action=disable'>On</A><br>")
+	else
+		dat += text("Exporter: <A href='?src=\ref[src];action=enable'>Off</A><br>")
+	dat += text("Power consumption: <A href='?src=\ref[src];action=set_power'>[drain_rate] kilowatts</A><br>")
+	dat += text("Surplus power: [(powernet == null ? "Unconnected" : "[powernet.netexcess/1000] kilowatts")]<br>")
+	dat += text("Power exported: [power_drained] kilowatts<br>")
+	dat += text("Points earned from exports: [power_drained/200] points<br>")
+	switch(drain_rate)
+		if(0 to 200)
+			rating = "TERRIBLE"
+		if(201 to 400)
+			rating = "BAD"
+		if(401 to 800)
+			rating = "SUBPAR"
+		if(801 to 2500)
+			rating = "DECENT"
+		if(2501 to 6000)
+			rating = "ROBUST"
+		if(6001 to 10000)
+			rating = "GOOD BOY"
+		if(10001 to 9999999)
+			rating = "HOLY SHIT YOU ABSOLUTE MADMAN!"

```

Comment:
Leaderboards and stat tracking are staples of any game. We're not so sophisticated and yet we're always fostered a lackadaisical atmosphere, this is a simple way for players to get an idea of where their efforts fall.

I'll take that any day over most departments where the only measure of success is how much of a body count you can rack up after you finish your routine.